### PR TITLE
Fixes mood healing being comically large

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

bad argument pass

:cl:  
bugfix: mood is no longer the strongest healing chemical in the game
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -56,9 +56,9 @@
 		if(moody && moody.mood_level >= 7) // heal 0.2hp per second if you have 7 or more mood(I feel pretty good)
 			if(prob(50))
 				if(prob(50))
-					heal_bodypart_damage(0.2*seconds, 0, 0, TRUE, BODYPART_ORGANIC)
+					heal_bodypart_damage(0.2, 0, 0, TRUE, BODYPART_ORGANIC)
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/human/life.dm]:
```suggestion
					heal_bodypart_damage(0.2*2, 0, 0, TRUE, BODYPART_ORGANIC)
```

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -56,9 +56,9 @@
...
 				else
-					heal_bodypart_damage(0, 0.2*seconds, 0, TRUE, BODYPART_ORGANIC)
+					heal_bodypart_damage(0, 0.2, 0, TRUE, BODYPART_ORGANIC)
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/human/life.dm]:
```suggestion
					heal_bodypart_damage(0, 0.2*2, 0, TRUE, BODYPART_ORGANIC)
```

**Review by Bibby0110 — DISMISSED** [review]
To restore pre-bug numbers

**Review by ThatLing — CHANGES_REQUESTED** [review]
Implement delta_time properly in ``/datum/controller/subsystem/mobs/fire`` instead

**Comment by Bibby0110** [comment]
> Implement delta_time properly in `/datum/controller/subsystem/mobs/fire` instead

Not needed, delta time is just * 2

**Review by ynot01 — APPROVED** [review]

**Review by ThatLing — APPROVED** [review]
I will port the ``delta_time`` for mobs at a later point

**Pull request merged**